security.go

```diff
@@ -79,6 +81,17 @@ func (v *SecretsVerifier) Write(body []byte) (n int, err error) {
 	return v.hmac.Write(body)
 }
 
+// WriteBody will write the contents of the request body to the HMAC digest. The request body is then replaced with a new reader
+// so that future uses of the request body will succeeed
+func (v *SecretsVerifier) WriteBody(r *http.Request) (int, error) {
```

Comment:
io.Copy does this for you.

Reply:
It does one portion of this, writing the bytes to the verifier but it does not replace the request body with a new buffer so that it can be read again later. I have tested this locally and io.Copy consumes the request body and an empty slash command is returned afterwards. 

The unit test I have written shows that the same body can be read again (for slash command parsing) after request verification.

Reply:
using either io.MultiWriter or io.TeeReader resolves these problems, regardless this method is unnecessary. happy for a more up to date example but not extraneous code. =)

Reply:
just to make it explicit:

```
r.Body = io.TeeReader(r.Body, verifier)
if cmd, err = slack.SlashCommandParse(r); err != nil {}
if err = verifier.Ensure(); err != nil {}
```

Reply:
[the original issue](https://github.com/nlopes/slack/issues/353) has a whole discussion around the Write method and the io.Tee etc.

Reply:
ah, that makes sense. thanks for clarifying. updated the example to use a TeeReader.